src/slurm_plugin/slurm_resources.py

```diff
@@ -209,6 +212,10 @@ def is_poweing_up_idle(self):
     def is_ice(self):
         return self.error_code in self.EC2_ICE_ERROR_CODES
 
+    def is_correctly_power(self):
```

Comment:
not a huge fun of this name. we might even drop this function and have the caller due the boolean logic if we can't find a method that makes sense for the node state itself but it's more related to the logic being applied by the caller.

Reply:
Removed the function